includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -1726,12 +1725,12 @@ public function get_level3_data_from_order( string $merchant_country, WC_Order $
 				'discount_amount'     => $discount_amount, // The amount an item was discounted—if there was a sale,for example, as a non-negative integer.
 			];
 		};
-		$items_to_send = array_map( $process_item, $order_items );
 
 		$level3_data = [
 			'merchant_reference' => (string) $order->get_id(), // An alphanumeric string of up to  characters in length. This unique value is assigned by the merchant to identify the order. Also known as an “Order ID”.
+			'customer_reference' => (string) $order->get_id(),
```

Comment:
For my understanding:

From: https://github.com/Automattic/woocommerce-payments/issues/1962#issuecomment-876758134
>customer_reference. It is the only field in the docs we don't include in the plugin level3 handling. I checked with Karl/Stripe and he assures us we can ignore that field.

A bit confused from the comments, do we need to include this field? as Allen suggested we can ignore it.

Reply:
We definitely should, as it's a reference for buyers.